build.sbt

```diff
@@ -6,11 +6,19 @@ ThisBuild / scalacOptions ++= Seq(
   "-feature",
   "-unchecked",
   "-Xlint",
-  "-Xfatal-warnings",
   "-Xsource:3"
 )
+ThisBuild / scalacOptions ++= sys.props
+  .get("lerna.enable.discipline")
+  .map(_ => "-Xfatal-warnings")
+  .toSeq
 ThisBuild / autoAPIMappings := true
 
+// Minimum Coverage
+ThisBuild / coverageFailOnMinimum := true
+ThisBuild / coverageMinimumStmtTotal := 80
+ThisBuild / coverageMinimumBranchTotal := 80
```

Comment:
NOTE: These settings can be found [here](https://github.com/scoverage/sbt-scoverage#minimum-coverage).